src/test/groovy/net/fabricmc/loom/test/unit/ArtifactMetadataTest.groovy

```diff
@@ -158,9 +159,35 @@ class ArtifactMetadataTest extends Specification {
 		"1.4"       | "2.4"
 	}
 
-	def "Accepts all Loom versions"() {
+	def "Accepts all Loom versions for remap 'false'"() {
 		given:
-		def zip = createMod(modLoomVersion, "static")
+		def zip = createModWithRemap(modLoomVersion, false)
+		when:
+		def metadata = createMetadata(zip, loomVersion)
+		then:
+		metadata != null
+		where:
+		loomVersion | modLoomVersion
+		// Valid
+		"1.4"       | "1.0.1"
+		"1.4"       | "1.0.99"
+		"1.4"       | "1.4"
+		"1.4"       | "1.4.0"
+		"1.4"       | "1.4.1"
+		"1.4"       | "1.4.99"
+		"1.4"       | "1.4.local"
+		"1.5"		| "1.4.99"
+		"2.0"		| "1.4.99"
```

Comment:
This should be all spaces, no tabs since it's for alignment:
![](https://github.com/user-attachments/assets/765caabd-8a14-4aac-9faf-b29f9a5360f2)